### PR TITLE
Kevin/site perf2

### PR DESCRIPTION
**Pull request by danlipert**
##### Description

Compressed images for site performance

##### Refers/Fixes

https://github.com/gitcoinco/web/pull/4791

### CODE REVIEW

**Comment by owocki** [comment]
you got the PR to open!  it was so big github's UI choked on the PR creation page for me :)

**Comment by danlipert** [comment]
> you got the PR to open! it was so big github's UI choked on the PR creation page for me :)

hah! yeah I had to try it a few times

File: app/assets/v2/css/gitcoin.css [hunk 1]
```diff
@@ -374,7 +374,6 @@ div.button-pink {
 }
 
 .footer__main {
-  background-image: url('/static/v2/images/footer.png');
```
Comment (thelostone-mc) [inline app/assets/v2/css/gitcoin.css]: Is this declared the right way elsewhere ?

File: app/assets/v2/css/home.css [hunk 2]
```diff
@@ -224,7 +224,7 @@ img.play {
   cursor: pointer;
   --padding: 40px;
   background-color: var(--gc-green-hover);
-  background-image: url('../images/home/bg-video-preview.svg');
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css]: same doubt as above ?

File: app/assets/v2/css/gitcoin.css (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/css/gitcoin.css]: Oh nevermind, we load it on scroll! Derp derp

**Comment by thelostone-mc** [comment]
@danlipert / @octavioamu you guys wanna sit and resolve all these before next deploy?

**Comment by owocki** [comment]
we really only need the image changes from this PR.. not the html changes (which are managed on https://github.com/gitcoinco/web/pull/4826 already )

**Comment by thelostone-mc** [comment]
@danlipert / @octavioamu So spent an annoying 1 hour resolving conflicts and ensuring only img compression happen

@owocki I've removed the compress + other html changes in this PR 

This PR now has 
- updating src to data-src
- compression of jquery-viewer file + all imgs

File: app/retail/templates/shared/footer.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
 {% endcomment %}
 {% load i18n static %}
 <footer class="container-fluid no-gutters footer">
-  <div class="container footer__main">
+  <div class="container footer__main" data-src="{% static "v2/images/footer.png" %}">
```
Comment (octavioamu) [inline app/retail/templates/shared/footer.html]: how this work?

Comment (owocki) [reply app/retail/templates/shared/footer.html]: jquery-unveil.js loads it iff the user scrolls down to the footer (not upon pageload)

**Comment by thelostone-mc** [comment]
Closing this out and opening this up against master

**Pull request closed without merging**